Compilation problem

## Bug description

Hi. I am trying to compile but I've got the following error:
```
[...]
  = note: /usr/bin/ld: /usr/src/cec-alsa-sync/target/debug/deps/libcec_rs-60b5991009b004f6.rlib(cec_rs-60b5991009b004f6.cec_rs.2b43c085-cgu.2.rcgu.o): in function `cec_rs::CecConnectionCfg::open':
          /home/manio/.cargo/registry/src/github.com-1ecc6299db9ec823/cec-rs-3.0.0/src/lib.rs:1201: undefined reference to `libcec_enable_callbacks'
          collect2: error: ld returned 1 exit status
          
  = help: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)

warning: `cec-alsa-sync` (bin "cec-alsa-sync") generated 3 warnings
error: could not compile `cec-alsa-sync` due to previous error; 3 warnings emitted
```
## Environment

- OS: Arch Linux

Is it a problem of too new libcec? Currently in arch it is `6.0.2-1`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 2.0.0
+
+- libcec-sys update. Now support libcec version 4, 5 and 6.
+
 ## 1.2.1
 
 - libcec-sys patch update

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4,7 +4,7 @@ path = 'src/bin.rs'
 
 [package]
 name = 'cec-alsa-sync'
-version = "1.2.2-alpha.0"
+version = "2.0.0-alpha.0"
 authors = ['Sami Salonen']
 edition = '2018'
 license = 'GPL-2.0'
@@ -18,7 +18,7 @@ homepage = 'https://github.com/ssalonen/cec-alsa-sync'
 repository = 'https://github.com/ssalonen/cec-alsa-sync'
 
 [dependencies]
-cec-rs = '3.0.0'
+cec-rs = '4.0.0'
 env_logger = '0.9.0'
 log = '0.4.11'
 

